### DISCUSSION TITLE
infrastructure/synology/ssh-with-rsa-key/

### DISCUSSION THREAD

**Discussion by giscus** [fire1ce/3os.org#95] [General]
# infrastructure/synology/ssh-with-rsa-key/

Synology DSM - Allow presistent SSH With RSA Keys. How to configure Synology to use RSA keys for SSH access.

https://3os.org/infrastructure/synology/ssh-with-rsa-key/

**Comment by pythoninthegrass** [comment]
Thanks for writing this up! I've had my NAS for at least a year and entering my ssh password was driving me 🍌

**Reply by fire1ce** [reply]
You are welcome!
